dashboard/config/database.yml

```diff
@@ -39,7 +39,7 @@ mysql_defaults: &mysql_defaults
   # You can set the value so that a lost connection can be detected earlier than the TCP/IP Close_Wait_Timeout value of 10 minutes.
   #
   # NOTE: mysql2 reuses this variable as a query timeout without retry, so set it to the maximum query execution time.
-  read_timeout: 30
+  read_timeout:  <%= unless ENV['db_read_timeout'].nil && ENV['db_read_timeout'] > 0 ? ENV['db_read_timeout']   : 30 %>
```

Comment:
We might be able to do something like:

``` ruby
read_timeout: ENV['DB_READ_TIMEOUT'] || 30
```

Reply:
I didn't know which one would be the right syntax :p 

Thanks!

Reply:
I wasn't sure if the lack of value is nil or zero. Because if it is zero it would make it have no limit